template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Program.cs

```diff
@@ -16,7 +16,7 @@ public static void Main(string[] args)
             var host = new WebHostBuilder()
                 .UseKestrel()
                 .UseContentRoot(Directory.GetCurrentDirectory())
-                .ConfigureConfiguration((context, builder) => {
+                .ConfigureAppConfiguration((context, builder) => {
```

Comment:
I didn't find where the hosting package version were specified but they should be 2.0.0-*. I also assumed that if you are seeing build errors because of this update in Hosting that you are grabbing the newest packages.